src/planwise/client/sources/handlers.cljs

```diff
@@ -48,7 +48,7 @@
    (rf/dispatch [:modal/hide])
    (-> db
        (dissoc :new)
-       (update :list asdf/swap! into [created-source]))))
+       (update :list #(asdf/swap! % conj created-source)))))
```

Comment:
The anonymous function is not needed here, since the ordering of the arguments does not change.